flink-architecture-tests/flink-architecture-tests-test/src/main/java/org/apache/flink/architecture/rules/ITCaseRules.java

```diff
@@ -126,6 +126,11 @@ public class ITCaseRules {
                                                             // Please refer to FLINK-25858.
                                                             .or(
                                                                     miniClusterWithClientResourceClassRule())
+                                                            .or(
+                                                                    JavaClass.Predicates
+                                                                            .assignableTo(
+                                                                                    AbstractTestBase
+                                                                                            .class))
```

Comment:
This should be covered by the miniClusterExtensionRule() thing; if it isn't then `containAnyFieldsInClassHierarchyThat` doesn't work as expected.

Reply:
Good catch, this change is unnecessary I reverted it.